test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs

```diff
@@ -153,8 +120,7 @@ public async Task UninstallAsync(CancellationToken token)
                     async () =>
                     {
                         string[] output =
-                            await Process.RunAsync("apt-get", "purge --yes libiothsm-std iotedge", token);
-
+                            await Process.RunAsync($"{this.packageManagement.PackageTool}", $"{this.packageManagement.UninstallCmd} libiothsm-std iotedge", token);
```

Comment:
Given my other comment about the problems with these `PackageManagement` properties, I'd create a `PackageManagement.GetUninstallCommandsAsync` method instead. Also, a `PackageManagement.GetIotedgeServicesAsync` method.

Reply:
How about a property that has a private set; but public get;

Reply:
A property can't be async so you'd still have the problem of mixing sync and async code.

Reply:
Yeah.. I revisit my approach there.
For the properties that required outside of the class, I create a getter for them.